index.html

```diff
@@ -48,7 +57,180 @@ <h1>I’m Nurbol<br>Glad to see<br> you!</h1>
+                    </div>
+            </div>
+
+<!--2 work card-->   
+
+            <div class="works-card">
+                <img src="worksimg\Snapshoot Portfolio@2x (1).png">
+                <div class="info-block">
+                    <div class="top">
+                        <h2 id="eight">Multi-Post<br>Stories</h2>
+                        <ul class="frame">
+                            <li id="client">CANOPY</li>
+                            <li id="dot"> </li>
+                            <li id="role">Back End Dev</li>
+                            <li id="dot"> </li>
+                            <li id="year">2015</li></ul>
+                    </div>
+                        <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                <div class="bottom">
+                    <ul class="tags">
+                    <li>html</li>
+                    <li>css</li>
+                    <li>javaScript</li></ul>
+                    <button class="action">See Project</button>
+                </div>
+            </div>
+        </div>
+
+<!--3 work card-->
+
+<div class="works-card">
+    <img src="worksimg/Snapshoot Portfolio@2x (2).png">
+    <div class="info-block">
+        <div class="top">
+            <h2 id="six">Tonic</h2>
+            <ul class="frame">
+                <li id="client">CANOPY</li>
+                <li id="dot"> </li>
+                <li id="role">Back End Dev</li>
+                <li id="dot"> </li>
+                <li id="year">2015</li></ul>
+        </div>
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        <div class="bottom">
+            <ul class="tags">
+            <li>html</li>
+            <li>css</li>
+            <li>javaScript</li></ul>
+            <button class="action">See Project</button>
+        </div>
+    </div>
+</div>
+
+<!--4 work card-->
+
+    <div class="works-card">
+        <img src="worksimg/Snapshoot Portfolio@2x (3).png">
+        <div class="info-block">
+            <div class="top">
+            <h2 id="eight">Multi-Post<br>Stories</h2>
+            <ul class="frame">
+                <li id="client">CANOPY</li>
+                <li id="dot"> </li>
+                <li id="role">Back End Dev</li>
+                <li id="dot"> </li>
+                <li id="year">2015</li></ul>
+            </div>
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <div class="bottom">
+                <ul class="tags">
+                <li>html</li>
+                <li>css</li>
+                <li>javaScript</li></ul>
+                <button class="action">See Project</button>
+            </div>
+        </div>
+    </div>            
+</div>    
+    </section>
+
+<!-- ─── About Me Section ─── -->    
+
+    <section class="aboutme">
+        <div class="text">
+            <h1>About<br>Myself</h1>
```

Comment:
- [ ] Having more than one `h1` tag in a single document is not a conventional practice and it is not good for SEO ranking. All of them will have to compete for significance in describing what the site is about. How about using other `heading` tags other than this? 😉